molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/controller/MappingServiceController.java

```diff
@@ -120,8 +137,6 @@ public String addMappingProject(@RequestParam("mapping-project-name") String nam
 			@RequestParam("target-entity") String targetEntity)
 	{
 		MappingProject newMappingProject = mappingService.addMappingProject(name, getCurrentUser(), targetEntity);
-		// FIXME need to write complete URL else it will use /plugin as root and the molgenis header and footer wont be
-		// loaded
 		return "redirect:/menu/main/mappingservice/mappingproject/" + newMappingProject.getIdentifier();
```

Comment:
A forward is an internal redirect, so I don't understand. You can just forward to /plugin/mappingservice/etcetc. Is this what you tried to do?